packages/cozy-stack-client/src/FileCollection.js

```diff
@@ -386,6 +388,14 @@ class FileCollection extends DocumentCollection {
    * @param {File|Blob|string|ArrayBuffer} attributes.data Will be used as content of the created file
    */
   async create(attributes) {
+    if (getIllegalCharacters(attributes).length) {
+      throw new Error(
+        `Invalid filename containing illegal character(s): ${getIllegalCharacters(
+          attributes
+        )}`
+      )
+    }
```

Comment:
Since we can throw, Is there any point in adding a @throw property to the JSDoc?

Reply:
I updated the whole file, thx